src/ga4gh/cat_vrs/profile_models.py

```diff
@@ -0,0 +1,154 @@
+"""Define Pydantic models for GA4GH categorical variation objects.
+
+See the `CatVar page <https://www.ga4gh.org/product/categorical-variation-catvar/>`_ on
+the GA4GH website for more information.
+"""
+
+from enum import Enum
+
+from ga4gh.cat_vrs.core_models import (
+    CategoricalVariant,
+    Constraint,
+    CopyChangeConstraint,
+    CopyCountConstraint,
+    DefiningContextConstraint,
+    Relation,
+)
+from pydantic import BaseModel, Field, field_validator
+
+
+class CatVrsType(str, Enum):
```

Comment:
So this is loss of a problem and more of a thin got keep in mind for later, there are an astronomically large number of possible CatVar types.  What we represent in `recipes` is only a very small subset of that, and do not constitute limits on valid CatVars, they just serve to define common useful CatVar classes of particular interest to the implementors.